index.html

```diff
@@ -158,62 +187,62 @@ <h1>
       </section>
     </section>
```

Comment:
- [ ] kindly consider removing the height and width on these two buttons `get my resume and get in touch ` to make your design an exact duplicate of the provided design  check the images below for more clarification and as you do that make sure that the mobile version is not affected as well
- your design 👇 
<img width="867" alt="Screen Shot 2023-05-10 at 08 31 39" src="https://github.com/mzirkof/R-PortFolio/assets/60146030/7a743a6e-56c4-4f90-ad40-4a7ceb08aa10">


- provided design 👇 
<img width="537" alt="Screen Shot 2023-05-10 at 08 32 16" src="https://github.com/mzirkof/R-PortFolio/assets/60146030/3e3c2d15-03eb-414b-9811-afff17e7925d">

Reply:
Ok done